rules/__tests__/lowercase_description.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rules = require('../..').rules;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+  },
+});
+
+const errMessage = 'it() description should begin with lowercase';
+const errors = [{ message: errMessage, column: 1, line: 1 }];
+
+ruleTester.run('lowercase-description', rules['lowercase-description'], {
+  valid: [
+    "it('foo', function () {})",
+    'it("foo", function () {})',
+    'it(`foo`, function () {})',
+    'it("<Foo/>", function () {})',
+    'it("123 foo", function () {})',
+    'describe("foo", function () {})',
+    'test("foo", function () {})',
+  ],
+
+  invalid: [
```

Comment:
I think it should warn for `describe` and `it` as well

Reply:
I agree, I think that'd be nice!